zetaclient/chains/evm/signer/gas.go

```diff
@@ -10,10 +10,7 @@ import (
 	"github.com/zeta-chain/node/x/crosschain/types"
 )
 
-const (
-	minGasLimit = 100_000
-	maxGasLimit = 1_000_000
-)
+const maxGasLimit = 2_500_000
```

Comment:
would be great if you could put some comment here, for example: `maxGasLimit` is the maximum gas limit cap for EVM chain outbound to prevent excessive gas